AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -374,6 +380,20 @@ - (void)sendLog:(id<MSLog>)log flags:(MSFlags)flags {
   [self.channelUnit enqueueItem:log flags:flags];
 }
 
+- (void)setTransmissionInterval:(NSUInteger)interval {
+  if (interval > kMSFlushIntervalMaximum || interval < kMSFlushIntervalMinimum) {
+    MSLogWarning([MSAnalytics logTag],
+                 @"The transmission interval is invalid, it should be between 3 second and 1 day (86400 seconds).");
```

Comment:
We decided to use "The transmission interval is not valid.". Please revise.
Also the log should be an error level.